LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2024 Florian Lefebvre
```

Comment:
I'm fine with having you here first, but I would also like to have the org. What I did in the past was adding either a second line or comma separated e.g. `Florian Lefebvre, Astrolicious Members`

Reply:
Just found two more examples in Astro:

https://github.com/withastro/astro/blob/main/LICENSE#L26
https://github.com/withastro/astro/blob/main/LICENSE#L40

Reply:
yeah I like the `<author> and Astrolicious contributors`. Will update it! As for being a bit more protective with the content, we can update the license at any time for sure! There are a few topics to discuss first (such as cross posting)